packages/workbox-background-sync/demo/sw.js

```diff
@@ -1,8 +1,8 @@
 /* global goog*/
 importScripts(
 	'../build/background-sync-queue.js',
-	'../../sw-routing/build/sw-routing.js',
-	'../../sw-runtime-caching/build/sw-runtime-caching.js'
+	'../../workbox-routing/build/workbox-routing.js',
+	'../../workbox-runtime-caching/build/workbox-runtime-caching.js'
```

Comment:
At the moment this changes the package names and fixes build. Namespace changes I'd do in a seperate PR (just to keep this PR from ballooning further).